odoo/addons/base/models/res_users.py

```diff
@@ -1188,7 +1188,7 @@ def _update_user_groups_view(self):
                 # group of res.groups without implied groups (with each other).
                 if app.xml_id == 'base.module_category_user_type':
                     # application name with a selection field
-                    field_name = name_selection_groups(gs.ids)
+                    field_name = name_selection_groups(gs.sorted('id').ids)
```

Comment:
I guess this would be equivalent:
```suggestion
                    field_name = name_selection_groups(sorted(gs.ids))
```

Reply:
Indeed. Cleaner :slightly_smiling_face: :+1: